redaxo/src/core/lib/rex.php

```diff
@@ -131,15 +131,35 @@ public static function setProperty($key, $value)
      * @throws InvalidArgumentException on invalid parameters
      *
      * @return mixed The value for $key or $default if $key cannot be found
+     * @psalm-return (
+     *     $key is 'login' ? rex_backend_login :
```

Comment:
```suggestion
     *     $key is 'login' ? rex_backend_login|null :
```

Im Frontend ist es null.

Reply:
Hier hast du null wieder entfernt?

Reply:
hmm bad merge.. ist jetzt wieder drinn